spec/lib/stripe/payment_intent_validator_spec.rb

```diff
@@ -38,14 +66,41 @@ module Stripe
       end
 
       context "when payment intent contains an error" do
+        let!(:pm_card) do
+          Stripe::PaymentMethod.create({
+                                         type: 'card',
+                                         card: {
+                                           # decline code: insufficient_funds
+                                           number: '4000000000009995',
+                                           exp_month: 12,
+                                           exp_year: 2034,
+                                           cvc: '314',
+                                         },
+                                       })
+        end
+
+        let!(:payment_intent) do
+          Stripe::PaymentIntent.create({
+                                         amount: 100,
+                                         currency: 'eur',
+                                         payment_method: pm_card,
+                                         payment_method_types: ['card'],
+                                         capture_method: 'manual',
+                                       })
+        end
+
         let(:payment_intent_response_body) {
           JSON.generate(id: payment_intent_id, last_payment_error: { message: "No money" })
         }
 
         it "raises Stripe error with payment intent last_payment_error as message" do
+          expect {
+            Stripe::PaymentIntent.confirm(payment_intent.id)
+          }.to raise_error Stripe::StripeError, "Your card has insufficient funds."
+
           expect {
             validator.call
-          }.to raise_error Stripe::StripeError, "No money"
+          }.to raise_error Stripe::StripeError, "Your card has insufficient funds."
```

Comment:
Great! Now we can actually re-record this after any stripe updates and check if the specs still pass. This is huge!